packages/kbn-unified-data-table/src/components/custom_control_columns/additional_row_control/row_control_column.tsx

```diff
@@ -37,28 +37,39 @@ export const RowControlCell = ({
         label,
         onClick,
         tooltipContent,
+        ...extraProps
       }) => {
-        return (
-          <EuiToolTip
-            content={tooltipContent ?? label}
-            delay="long"
-            anchorClassName="unifiedDataTable__rowControl"
-          >
-            <EuiButtonIcon
-              data-test-subj={dataTestSubj ?? `unifiedDataTable_rowControl_${props.columnId}`}
-              disabled={disabled}
-              iconSize="s"
-              iconType={iconType}
-              color={color ?? 'text'}
-              aria-label={label}
-              onClick={() => {
-                if (record) {
-                  onClick?.({ record, rowIndex });
-                }
-              }}
-            />
-          </EuiToolTip>
+        const control = (
+          <EuiButtonIcon
+            aria-label={label}
+            className="unifiedDataTable__rowControl"
+            color={color ?? 'text'}
+            data-test-subj={dataTestSubj ?? `unifiedDataTable_rowControl_${props.columnId}`}
+            disabled={disabled}
+            iconSize="s"
+            iconType={iconType}
+            onClick={() => {
+              if (record && onClick) {
+                onClick({ record, rowIndex });
+              }
+            }}
+            {...extraProps}
+          />
         );
+
+        if (tooltipContent) {
+          return (
+            <EuiToolTip
+              anchorClassName="unifiedDataTable__rowControl"
```

Comment:
`unifiedDataTable__rowControl` is doubled now. We can either fix it or revert changes.

Reply:
Addressed with [refactor(discover): change requests](https://github.com/elastic/kibana/pull/195630/commits/5546bf2fdde40582f687cdce9ca4d87b266cb11d), now either the button alone or the tooltip wrapper will get the classname.